localstack/services/sqs/legacy/packages.py

```diff
@@ -0,0 +1,36 @@
+import os
+from typing import List
+
+from localstack.packages import InstallTarget, Package, PackageInstaller
+from localstack.services.install import log_install_msg
+from localstack.utils.files import mkdir
+from localstack.utils.http import download
+
+ELASTICMQ_JAR_URL = (
+    "https://s3-eu-west-1.amazonaws.com/softwaremill-public/elasticmq-server-1.1.0.jar"
+)
+
+
+class ElasticMQPackage(Package):
+    def __init__(self):
+        super().__init__("ElasticMQ", "1.1.0")
+
+    def get_versions(self) -> List[str]:
+        return ["1.1.0"]
+
+    def _get_installer(self, version: str) -> PackageInstaller:
+        return ElasticMQPackageInstaller("elasticmq", version)
+
+
+class ElasticMQPackageInstaller(PackageInstaller):
+    def _get_install_marker_path(self, install_dir: str) -> str:
+        return os.path.join(install_dir, "elasticmq-server.jar")
+
+    def _install(self, target: InstallTarget) -> None:
+        log_install_msg("ElasticMQ")
+        install_dir = self._get_install_dir(target)
+        mkdir(install_dir)
+        download(ELASTICMQ_JAR_URL, self._get_install_marker_path(self._get_install_dir(target)))
```

Comment:
You could just use the `DownloadInstaller` here.

Reply:
yes I agree, but tbh spending time on this installer at all was a mistake, since elasticmq is not really used anymore and I suspect it will be removed soonish. So when I realised that I could have used the download installer for this, I decided to not waste any more time on semi-dead code.